lib/spack/env/cc

```diff
@@ -524,7 +533,8 @@ fi
 
 # dump the full command if the caller supplies SPACK_TEST_COMMAND=dump-args
 if [[ $SPACK_TEST_COMMAND == dump-args ]]; then
-    echo "${full_command[@]}"
+    IFS="
+" && echo "${full_command[*]}"
```

Comment:
Does some `bash` expert know a better way to do this?  The newline is nasty but I couldn't get a quoted `"\n"` to work properly.